src/qt/bitcoingui.cpp

```diff
@@ -59,6 +59,7 @@
 #include <QToolButton>
 #include <QUrlQuery>
 #include <QVBoxLayout>
+#include <QWindow>
```

Comment:
please explain how 14573 is partial

Reply:
My mistake, it's not. I can edit the commit message now or later before squashing

Reply:
Up to you :)